pkg/network/bootstrap.go

```diff
@@ -0,0 +1,19 @@
+package network
+
+import (
+	"github.com/pkg/errors"
+
+	netv1 "github.com/openshift/cluster-network-operator/pkg/apis/networkoperator/v1"
+)
+
+// Bootstrap creates resources required by SDN on the cloud.
+func Bootstrap(conf *netv1.NetworkConfigSpec) (map[string]interface{}, error) {
```

Comment:
Bootstrap should definitely return a properly typed struct. 

It also should reconcile idempotently against OpenStack (or whatever), creating or updating resources as required.

It should also commit its changes - the correct way to do this would be the `NetworkConfig.Status` field, but I could be convinced to store it just as part of the Applied configuration.

Reply:
Done.